CMakeLists.txt

```diff
@@ -18,9 +18,9 @@ OPTION(USE_TSAN "Use ThreadSanitizer (clang)" OFF)
 OPTION(USE_REDIS_ALLOCATOR "Use redis allocator" ON)
 OPTION(USE_COVERAGE "Build with coverage" OFF)
 
-IF (RS_MODULE_NAME) 
-    ADD_DEFINITIONS("-DREDISEARCH_MODULE_NAME=\"${RS_MODULE_NAME}\"")
-ENDIF()
+# IF (RS_MODULE_NAME) 
```

Comment:
@rafie if think we need it for the light build no?

Reply:
Yes, I already fixed it. You know how much I love the way we build Light, right? :-)